koji/flatten-el8/script.sh

```diff
@@ -0,0 +1,10 @@
+dnf install -y epel-release
+dnf install -y ansible "dnf-command(reposync)"
+
+pushd /root/koji-sync/podman/repos
+dnf reposync --repoid AppStream --repoid PowerTools --repoid BaseOS --download-metadata
+
+pushd /root/koji-sync/podman/koji-ansible/
+ansible-playbook -i inventory/inventory master.yaml --connection=local
```

Comment:
If you're doing to use podman, wouldn't it make sense to write a `Dockerfile` to install all the tools so you can cache the image?

Reply:
Yes! But that would require me to learn how to write a Dockerfile.  In the very short timeframe I had, i worked with what resources I knew to get it knocked out.

As I get time, i'll swing back to this and add one.

Reply:
It'd roughly be https://github.com/theforeman/foreman-infra/commit/105043aac95155f06733488a1d3fed2e8351f108

Reply:
That leaves two options in my mind.  a) merge current state "as is" then update it, then improve upon it, or b) Go ahead and do the revisions now for this pr.

I'm in favor of a, primarily because it captures what we are doing at this exact moment, in case of catastrophic failure, and then we can go back and improve upon it iteratively.

Reply:
Note that my commit is on top of this commit.